src/clarity_vm/clarity.rs

```diff
@@ -1082,6 +1082,10 @@ impl<'a, 'b> ClarityBlockConnection<'a, 'b> {
         })
     }
 
+    pub fn initialize_epoch_3_0(&mut self) -> Result<Vec<StacksTransactionReceipt>, Error> {
+        todo!(); // TODO(sbtc): Epoch 3.0 initialization logic
```

Comment:
If there's nothing to do here, then just return `Ok(vec![])`.

Reply:
Currently I don't know exactly what needs to be done here. I'll move this to a separate branch focused on Epoch 3.0.